docs/templates/redmine_template.txt.j2

```diff
@@ -1,5 +1,16 @@
 h1. EC-Earth 4: Monitoring
 
+h2. Contained Plots
```

Comment:
I mentioned this elsewhere, but just to have it all in one place: I would not use this headline. The toc is below the issue header in the actual Redmine page, and stands for itself. No extra hint needed that this is a toc, or rather a list of shortcut links.